### PR TITLE
Gives the bartender their own version of judo

### PR DESCRIPTION
**Pull request by GDNgit**
## What Does This PR Do
Gives the bartender their own version of judo, like chef CQC, it can only be used while in the bar/kitchen

## Why It's Good For The Game
So right now the bartender has rubbershot. Although it's quite a bit better than beanbags, I wouldn't call it nonlethal or not totally prone to hitting people you weren't aiming for. Giving them judo will give them a fun fully nonlethal option to play with, also if the chef can have CQC and it's fine worse baton should also be completely fine.  You can use a baton while not in the bar or the kitchen.

I also just like the idea of the chef and the bartender working together with martial arts to remove a tider from the kitchen/bar.

## Testing
Judo'd punpun until horizotal then batoned some people

## Changelog
:cl:
tweak: The bartender now has access to judo, only works in the bar or kitchen
/:cl:

### CODE REVIEW

**Comment by OctusGit** [comment]
Yes

**Comment by SpringSkipper** [comment]
The shotgun seems a lot lot more iconic to me. It really isn't lethal unless you're trying to be a shitter, and switching to judo won't stop that issue.

There's also the question of why bartender gets the worse martial art of the two.

**Comment by GDNgit** [comment]
> The shotgun seems a lot lot more iconic to me. It really isn't lethal unless you're trying to be a shitter, and switching to judo won't stop that issue.

They still get the shotgun, and it being iconic isn't a defense 

> There's also the question of why bartender gets the worse martial art of the two.

They get a shotgun

**Comment by SpringSkipper** [comment]
> They still get the shotgun, and it being iconic isn't a defense
Ah, they get **both**, okay. From your comment about the shotgun being less than ideal I thought it was a replacement.

Also yes, it being iconic is a defense. It's much better than a weird meme about a conditionally active brain chip that teaches you martial arts because an NNO got inconvenienced once. Bartenders having shotguns is a general thing in fiction and media.

It'd be like if someone PR'd in a replacement for the Ripley mech (iconic powerloader-type mech that makes sense in context) that was a mech which looks like optimus prime (silly meme thing that doesn't really make sense...)

CQC on chef is already weird enough; why are we handing out department-locked martial arts to random jobs to deal with tiders? Tiders are something that every job has to deal with to some extent. Are we going to see cargo-fu and science-kata soon?

**Comment by Coolrune206** [comment]
I'm not sure I like the Chef getting both a shotgun _and_ a form of CQC. This might be interesting as a "choice" item made by the Bartender at the start of the round, perhaps by picking up and interacting with something like a book called "Bar Defense and You" that either gives ol' trusty double barrel or the Bar-Judo. This'd also reinforce Judo as a "choice" thing, officers choosing it or the baton, bartenders choosing it or the shotty.

File: code/game/jobs/job/support.dm [hunk 1]
```diff
@@ -164,7 +164,8 @@
 	singlemutcheck(H, GLOB.soberblock, MUTCHK_FORCED)
 	H.dna.default_blocks.Add(GLOB.soberblock)
 	H.check_mutations = 1
-
+	var/datum/martial_art/judo/under_siege/bouncer_delight = new
+	bouncer_delight.teach(H)
```
Comment (Gaxeer) [inline code/game/jobs/job/support.dm:168]: do we really need to create new datum?

**Comment by tipoima** [comment]
If anything, I feel like Chef CQC is a meme that overstayed its welcome. Giving the bartender a different version of the same meme is just making bar hell for any antag unfortunate enough to have a target in it.
Personally, I'd instead opt into nerfing Chef's CQC to Judo and leaving the bartender with his free shotgun.

**Review by S34NW — APPROVED** [review]
deconflict for merge

**Pull request merged**